### DISCUSSION TITLE
Move Logs to Developer Tools

### DISCUSSION THREAD

**Discussion by joeonline** [home-assistant/frontend#13157] [Dashboard]
Since the YAML tab is now within the Developer Tools section, the Logs should also be a tab there too. This would enable quick access for troubleshooting issues from config changes on the same screen.

I am constantly clicking in between YAML and Logs to check if errors/warnings are fixed.

Even if its duplicated or the "raw/unformatted" logs remains at the current location.

**Comment by ildar170975** [comment]
I am using yaml tools & log on different constantly open browser's tabs.
If I need to update the log, I am pressing a corresponding "update" button.
